### DISCUSSION TITLE
Plot Size Filter to Match Theme

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#913] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/4793561/125228290-60f50780-e289-11eb-8b71-b90658e4946d.png)

It would be nice if either you could hid the plot size filter, it matched the general theme of the app, or you could specify the color. Here it is the only thing in the app that is blue

**Comment by mturoci** [comment, marked as answer]
Hi @mtanco, this should resolve this - https://github.com/h2oai/wave/pull/860.
